Core/Src/dti.c

```diff
@@ -252,57 +254,28 @@ int32_t dti_get_rpm(dti_t *mc)
 	return rpm;
 }
 
-/* Inbound Task-specific Info */
-osThreadId_t dti_router_handle;
-const osThreadAttr_t dti_router_attributes = {
-	.name = "DTIRouter",
-	.stack_size = 64 * 8,
-	.priority = (osPriority_t)osPriorityHigh
-};
+float dti_get_mph(dti_t *mc)
+{
+	/* Convert RPM to MPH */
+	// rpm * gear ratio = wheel rpm
+	// tire diamter (in) to miles --> tire diamter miles
+	// wheel rpm * 60 --> wheel rph
+	// tire diamter miles * pi --> tire circumference
+	// rph * wheel circumference miles --> mph
+	return (dti_get_rpm(mc) / (GEAR_RATIO)) * 60 *
+	       (TIRE_DIAMETER / 63360.0) * M_PI;
+}
 
-static void dti_record_rpm(dti_t *mc, can_msg_t msg)
+void dti_record_rpm(dti_t *mc, can_msg_t msg)
```

Comment:
just curious why not static?

Reply:
It was never exposed in the header file but now it is since its a callback in can_handler.

Reply:
It was not exposed in the header before, but now that it is it can't be static